src/test/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformerTest.java

```diff
@@ -2516,17 +2529,171 @@ public void givenNullFormWithOcrFormType_thenThrowNoError() {
+            "Other").build();
+
+        given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
+        CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
+
+        assertTrue(result.getErrors().isEmpty());
+        assertTrue(result.getWarnings().isEmpty());
+    }
+
+    @Test
+    public void givenNullFormTypeWithInputValidFormType_thenThrowNoError() {
+        prepareData("SSCS5");
+
+        ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
+            null).build();
+
+        given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
+        CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
+
+        assertTrue(result.getErrors().isEmpty());
+        assertTrue(result.getWarnings().isEmpty());
+    }
+
+    @Test
+    public void givenNullFormAndWithNullFormTypeInput_thenThrowError() {
+        prepareData(null);
 
         ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
             null).build();
+
         given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
+        CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
+
+
+        assertTrue(result.getErrors().size() == 1);
+        assertTrue(result.getWarnings().isEmpty());
+    }
+
+    @Test
+    public void givenValidFormAndWithNullFormTypeInput_thenNoThrowError() {
+        prepareData(null);
+
+        ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
+            "sscs5").build();
+
+        given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
         CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
 
+
         assertTrue(result.getErrors().isEmpty());
         assertTrue(result.getWarnings().isEmpty());
     }
 
+    @Test
+    public void givenOtherFormAndWithNullFormTypeInput_thenThrowError() {
+        prepareData(null);
+
+        ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
+            "Other").build();
+
+        given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
+        CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
+
+
+        assertTrue(result.getErrors().size() == 1);
+        assertTrue(result.getWarnings().isEmpty());
```

Comment:
Lines 2601-2610 can be extracted to function `assertOneError(String formType)` and also used in place of lines 2569-2578 above.